### PR TITLE
Add a chapter on ABEND Handling

### PR DESCRIPTION
**Pull request by tanto259**
Signed-off-by: Hartanto Ario Widjaya <tanto259@users.noreply.github.com>

This PR adds a chapter on ABEND handling, which includes common system ABEND and ABEND routine.

Contents are adapted from Jonathan Sayles.

Currently there are no labs for the chapter. Although one can be added where course takers would purposely execute programs that causes ABENDs and fix them.

### CODE REVIEW

File: COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md [hunk 1]
```diff
@@ -4299,3 +4299,210 @@ Previous lab programs made use of a date/time intrinsic function.  The date/time
 **Lab Hints**
 
 Refer to CBL0011 line 120 for the proper formatting of the function-name causing   the compile error.
+
+\newpage
+# ABEND handling
```
Comment (MikeBauerCA) [inline COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md:4304]: Please review how other chapters are set up with a brief introduction followed by a table of contents for the chapter.

Comment (tanto259) [reply COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md:4304]: Fixed in d52e3ff.

File: COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md [hunk 2]
```diff
@@ -4299,3 +4299,240 @@ Previous lab programs made use of a date/time intrinsic function.  The date/time
...
+- Incorrectly initialized or uninitialized variables
+- Missing or incorrect data edits
+- MOVE from a 01-level to a 01-level if the sending field is shorter than receiving field
+- MOVE of zeros to group-level numeric fields
+- Incorrect MOVE CORRESPONDING
+- Incorrect assignment statements when MOVE from one field to another
+
+### S0CB - Division by Zero
+
+Just like mathematic, attempting to divide a number with 0 in Enterprise COBOL is an undefined operation.
```
Comment (MikeBauerCA) [inline COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md:4452]: *like in mathematics

**Review by MikeBauerCA — APPROVED** [review]
Generally approve this PR, just need to resolve conflicts. Thanks!

File: COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md (hunk 2 above)
Comment (tanto259) [reply COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md]: Fixed in 76410de

**Pull request merged**